NBitcoin/Protocol/AddressManager.cs

```diff
@@ -1170,38 +1170,42 @@ internal void DiscoverPeers(Network network, NodeConnectionParameters parameters
 							CancellationToken = loopCancel,
 						}, p =>
 						{
-							CancellationTokenSource timeout = new CancellationTokenSource(TimeSpan.FromSeconds(5));
-							var cancelConnection = CancellationTokenSource.CreateLinkedTokenSource(timeout.Token, loopCancel);
-							Node n = null;
-							try
+							using(CancellationTokenSource timeout = new CancellationTokenSource(TimeSpan.FromSeconds(5)))
 							{
-								var param2 = parameters.Clone();
-								param2.ConnectCancellation = cancelConnection.Token;
-								var addrman = param2.TemplateBehaviors.Find<AddressManagerBehavior>();
-								param2.TemplateBehaviors.Clear();
-								param2.TemplateBehaviors.Add(addrman);
-								n = Node.Connect(network, p.Endpoint, param2);
-								n.VersionHandshake(cancelConnection.Token);
-								n.MessageReceived += (s, a) =>
+								using(var cancelConnection = CancellationTokenSource.CreateLinkedTokenSource(timeout.Token, loopCancel))

```

Comment:
you can put two `using` lines together as a start of a brace{} block, this way there would be no indentation changes and the diff would read much nicerly

Reply:
I mean this way: https://stackoverflow.com/a/9396092/544947

Reply:
Done.